pkg/sandbox/start.go

```diff
@@ -33,9 +36,11 @@ const (
 	taintEffect          = "NoSchedule"
 	sandboxContextName   = "flyte-sandbox"
 	sandboxDockerContext = "default"
-	k8sEndpoint          = "https://127.0.0.1:30086"
+	K8sEndpoint          = "https://127.0.0.1:6443"
+	sandboxK8sEndpoint   = "https://127.0.0.1:30086"
 	sandboxImageName     = "cr.flyte.org/flyteorg/flyte-sandbox"
-	demoImageName        = "cr.flyte.org/flyteorg/flyte-sandbox-lite"
```

Comment:
Should we gradually remove this support since it would break existing pipelines which rely on this image

Reply:
hmm, what do you mean?  Why would we need this?  I think having a whole new command would be even more confusing.  We're going to continue to build the image... cuz we don't want old versions of flytectl to break.  But once people upgrade to this version of flytectl they'll get the new version.

Reply:
Ok. So new version of flytectl will only support the new image because i see the exposed ports also changed which seem to be applicable only to new image